src/app/commands/cmd_new_brush.cpp

```diff
@@ -64,6 +64,10 @@ bool NewBrushCommand::onEnabled(Context* context)
 
 void NewBrushCommand::onExecute(Context* context)
 {
+  ASSERT(current_editor);
+  if (!current_editor)
+    return;
+
```

Comment:
Nice catch. I think this is a bigger problem. I think that several other commands could be affected by this kind of scenario (Home tab is selected, and we use a command that use the active document view or expected current_editor != nullptr when it isn't).

The real solution is to completely remove the `current_editor` global variable (maybe this is the older global variable in Aseprite, and the more postponed refactor that I've to do).

Anyway, I'll merge this PR, just one thing: Could you change the commit message to something like "Fix crash using NewBrush when Home tab is selected"